src/gsy_e/models/strategy/__init__.py

```diff
@@ -458,6 +459,15 @@ def __init__(self):
 
     parameters = None
 
+    @property
+    def simulation_config(self) -> SimulationConfig:
+        """Return the SimulationConfiguration used by the current market area.
+
+        NOTE: this configuration is currently unique and shared across all areas, but this might
+        change in the future (each area could have its own configuration).
```

Comment:
👍  This is why I would not vote for adding this property now. Leave it for now but I predict that we will revert it soonish.

Reply:
I agree that this will need to be changed. When this happens I suggest to just rename it to `area_config` or something similar. I wouldn't remove it completely: I think the property itself would still be useful and better than using `self.area.config`.